eksconfig/add-on-cluster-autoscaler.go

```diff
@@ -0,0 +1,54 @@
+package eksconfig
```

Comment:
This should be inside `AddOnNodeGroups`

Reply:
> This should be inside `AddOnNodeGroups`

This doesn't make sense anymore as a separate file if we are planning to take the route of embedding inside NodeGroup/ASG.